### DISCUSSION TITLE
Why making a new version instead of contributing directly to Antimicro repo?

### DISCUSSION THREAD

**Discussion by ooker777** [AntiMicroX/antimicrox#644] [General]
I suppose you have specific reason, but I don't see where it is

**Comment by pktiuk** [comment]
Original discussion about abandoning legacy version (instead of backporting fixes and changes) was here: https://github.com/AntiMicro/antimicro/issues/351

But long story short, there was legacy AntiMicro, which was abandoned  
Later there was fork called AntiMicroX created by juliagoda https://github.com/juliagoda/antimicrox/  
Then I came and decided to continue from the latest version of app.

**Reply by pktiuk** [reply]
Why she didn't contribute upstream? I do not remember.    
(But tbh I don't recommend [asking her](https://github.com/AntiMicroX/antimicrox/issues/34#issuecomment-724136350) directly)
